test/extensions/wasm/test_data/speed_cpp.cc

```diff
@@ -1,11 +1,97 @@
 // NOLINT(namespace-envoy)
 #include <string>
 
+#ifndef NULL_PLUGIN
 #include "proxy_wasm_intrinsics.h"
+#else
+#include "extensions/common/wasm/null/null_plugin.h"
+#endif
 
-extern "C" PROXY_WASM_KEEPALIVE void proxy_on_tick(uint32_t /* context_zero */) {
+START_WASM_PLUGIN(WasmSpeedCpp)
+
+int xDoNotRemove = 0;
```

Comment:
Why do we need this?

Reply:
This is to prevent the compiler from eliminating the code.